NEWS.md

```diff
@@ -41,6 +40,7 @@ kaiaulu [0.0.0.9600](https://github.com/sailuh/kaiaulu/milestone/5) (in developm
 ### DOCUMENTATION FIXES
 
  * New configuration file for the Apache Thrift project called "thrift.yml". [#148](https://github.com/sailuh/kaiaulu/issues/148)
+ * Adds unit tests for parse_mbox() and parse_jira(). [#154](https://github.com/sailuh/kaiaulu/issues/154)
```

Comment:
Leilani informed me that parse_mbox would no longer be tested since we'd need .mbox file. I can delete these tests if need.

Reply:
The parse_mbox() tests should not be deleted. You should use their logic on the parse_gitlog() combined to the function that creates a sample git log that I pointed out in the other comment.

Reply:
@carlosparadis Do you mean we would need to try and create a .mbox file on the fly and then delete it after the test?

Reply:
No. Instead of unit testing mbox, unit test the parse_gitlog() which we already have a function to create it on the fly. Say, if you are testing what happens if the path to a .mbox file passed doesn't exist in parse_mbox(), then transfer the logic to the parse_gitlog() by unit testing what happens if the path to the .git file is wrong. Same logic, different functions they are applied to.

Reply:
I see, so get rid of the parse_mbox tests and replace it with parse_gitlog tests, but using the same logic as the previous parse_mbox tests.

Reply:
Correct! If the logic of the unit tests is preserved by transferring to parse_gitlog(), I can always replicate it back down to parse_mbox() down the road when the generate on the fly tests for mbox are created. They are fundamentally parser functions at the end of the day.